commit c65f9a07a78afa3c98712f6192962ffd6babe339
Author: Greg Kurz <gkurz@linux.vnet.ibm.com>
Date:   Wed Dec 11 14:15:34 2013 +0100

    target-ppc: add stubs for KVM breakpoints
    
    The latest update to v3.13-rc3 (bf63839f) breaks the
    ppc build with KVM:
    
    kvm-all.o: In function `kvm_update_guest_debug':
    kvm-all.c:1910: undefined reference to `kvm_arch_update_guest_debug'
    kvm-all.o: In function `kvm_insert_breakpoint':
    kvm-all.c:1937: undefined reference to `kvm_arch_insert_sw_breakpoint'
    kvm-all.c:1945: undefined reference to `kvm_arch_insert_hw_breakpoint'
    kvm-all.o: In function `kvm_remove_breakpoint':
    kvm-all.c:1977: undefined reference to `kvm_arch_remove_sw_breakpoint'
    kvm-all.c:1985: undefined reference to `kvm_arch_remove_hw_breakpoint'
    kvm-all.o: In function `kvm_remove_all_breakpoints':
    kvm-all.c:2009: undefined reference to `kvm_arch_remove_sw_breakpoint'
    kvm-all.c:2006: undefined reference to `kvm_arch_remove_sw_breakpoint'
    kvm-all.c:2017: undefined reference to `kvm_arch_remove_all_hw_breakpoints'
    
    We need stubs until something gets implemented.
    
    Signed-off-by: Greg Kurz <gkurz@linux.vnet.ibm.com>
    Reviewed-by: Alexander Graf <agraf@suse.de>
    Signed-off-by: Alexander Graf <agraf@suse.de>

Index: qemu-2.0~git-20140225.aa0d1f4/target-ppc/kvm.c
===================================================================
--- qemu-2.0~git-20140225.aa0d1f4.orig/target-ppc/kvm.c	2014-02-25 22:37:35.978999996 -0600
+++ qemu-2.0~git-20140225.aa0d1f4/target-ppc/kvm.c	2014-02-25 22:37:35.970999996 -0600
@@ -1934,3 +1934,31 @@ void kvm_arch_remove_all_hw_breakpoints(
 void kvm_arch_update_guest_debug(CPUState *cpu, struct kvm_guest_debug *dbg)
 {
 }
+
+int kvm_arch_insert_sw_breakpoint(CPUState *cpu, struct kvm_sw_breakpoint *bp)
+{
+    return -EINVAL;
+}
+
+int kvm_arch_remove_sw_breakpoint(CPUState *cpu, struct kvm_sw_breakpoint *bp)
+{
+    return -EINVAL;
+}
+
+int kvm_arch_insert_hw_breakpoint(target_ulong addr, target_ulong len, int type)
+{
+    return -EINVAL;
+}
+
+int kvm_arch_remove_hw_breakpoint(target_ulong addr, target_ulong len, int type)
+{
+    return -EINVAL;
+}
+
+void kvm_arch_remove_all_hw_breakpoints(void)
+{
+}
+
+void kvm_arch_update_guest_debug(CPUState *cpu, struct kvm_guest_debug *dbg)
+{
+}
